code/game/alternate_appearance.dm

```diff
@@ -166,7 +166,7 @@ GLOBAL_LIST_EMPTY(active_alternate_appearances)
 			add_hud_to(mob)
 
 /datum/atom_hud/alternate_appearance/basic/blessed_aware/mobShouldSee(mob/M)
-	if(M.mind && (M.mind.assigned_role == "Chaplain"))
+	if(is_chaplain_job(M.mind?.assigned_role))
```

Comment:
it would change behaviour a tiny bit but this should use holy role checks instead of job checks

Reply:
There are a ton of small improvements to be made, but this PR already changes 186 files already and it's not the last one I'm planning on doing.
Most is-role checks should be substituted by checking if the property that job provides is present, instead, essentially. But that needs to be made atomically.